### ISSUE TITLE
Hello friends, I am a student from China. I made a one-handed glove, but I couldn't get feedback from ALEX. How can I solve it

### ISSUE THREAD

**Issue by Yangcrazy1** [LucidVR/lucidgloves#114]
Hello friends, I am a student from China. I made a one-handed glove, but I couldn't get feedback from ALEX. How can I solve it？and sometimes can't grasp the object

**Comment by MasterSiu** [comment]
You have to download plug in for the half life on lucid glove page

**Comment by Yangcrazy1** [comment]
> 您必须在清醒手套页面上下载半衰期插件

Yes, I downloaded and opened ALEX, but I only have one glove, can only grab objects but can't generate force feedback, what is the reason for this can be solved?

**Comment by Yangcrazy1** [comment]
923925220@qq.com 如果您了解怎么解决可以发邮件联系我

**Comment by Yang66g** [comment]
有点意思
